src/app/component/Select/Select.component.js

```diff
@@ -40,20 +41,23 @@ class Select extends Component {
      * @param {Object} option
      */
     renderSortOption(option) {
+        // TODO add onkeypress for options
         return (
                 <option
                   block="Select"
                   elem="Option"
                   key={ option.value }
                   value={ option.value }
+                  tabIndex={ 0 }
                 >
                 { option.label }
                 </option>
         );
     }
 
     render() {
-        const { options, selectedOption } = this.props;
+        // TODO add select as possible child type name in form component
+        const { options, selectedOption, formRef } = this.props;
```

Comment:
Passing formRef will not be enough. Please update Form to accept Select component inside-of it.